cassie/modules/empire.py

```diff
@@ -0,0 +1,340 @@
+import datetime
+
+from urllib.parse import urlparse
+
+from cassie.argparselite import ArgumentParserLite
+from cassie.imcontent import IMContentMarkdown
+from cassie.templates import CassieXMPPBotModule
+
+try:
+	import requests
+except ImportError:
+	print('Failed to import "requests" module.')
```

Comment:
This should never happen because the `requests>=2.8.1` is in the projects requirements file.